app/src/main/java/io/token/sample/Application.java

```diff
@@ -119,60 +112,61 @@ public static void main(String[] args) throws IOException {
             Map<String, String> formData = gson.fromJson(req.body(), type);
             String callbackUrl = req.scheme() + "://" + req.host() + "/redeem-standing-order-popup";
 
-            String tokenRequestUrl = initializeStandingOrderTokenRequestUrl(
-                    formData,
-                    callbackUrl,
-                    res);
+            String tokenRequestUrl =
+                    initializeStandingOrderTokenRequestUrl(formData, callbackUrl, res);
 
             // return the generated Token Request URL
             res.status(200);
             return tokenRequestUrl;
         });
 
-        // for redirect flow, use Token.parseTokenRequestCallbackUrl()
-        Spark.get("/redeem", (req, res) -> {
-            String callbackUrl = req.url() + "?" + req.queryString();
-
-            // retrieve CSRF token from browser cookie
-            String csrfToken = req.cookie(CSRF_TOKEN_KEY);
-
-            // check CSRF token and retrieve state and token ID from callback parameters
-            TokenRequestCallback callback = tokenClient.parseTokenRequestCallbackUrlBlocking(
-                    callbackUrl,
-                    csrfToken);
+        Spark.get("/one-step-payment", (req, res) -> {
+            Map<String, String> params = toMap(req.queryMap());
+            String callbackUrl = req.scheme() + "://" + req.host() + "/redeem-one-step-payment";
 
```

Comment:
@mahendra-aps in case of redeem-one-step-payment, it will be no-op as they don't do the redeem token but the token system does the redeem. So if we are calling redeem-one-step-payment then we should be no-op. can we make that change pls.

Reply:
done